packages/perennial/deploy/002_deploy_core.ts

```diff
@@ -74,16 +74,16 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // TIMELOCK OR CROSSCHAINOWNER
 
   // If this is mainnet, deploy a Timelock as ProxyAdmin owner
-  if (isEthereum(networkName)) {
+  if (isArbitrum(networkName)) {
```

Comment:
Are these changes meant to be permanent?

Reply:
oops - fixed